ci-operator/jobs/openshift/external-dns-operator/openshift-external-dns-operator-main-presubmits.yaml

```diff
@@ -0,0 +1,50 @@
+presubmits:
+  openshift/external-dns-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - main
+    cluster: build01
+    context: ci/prow/unit
+    decorate: true
+    decoration_config:
+      skip_cloning: true
```

Comment:
What is the intent of specifying `skip_cloning: true`?

Reply:
Let me ask :grin: